service/src/main/java/greencity/mapping/events/EventDtoMapper.java

```diff
@@ -30,34 +31,37 @@ public class EventDtoMapper extends AbstractConverter<Event, EventDto> {
 
     @Override
     public EventDto convert(Event event) {
-        List<EventDateDto> dates = new ArrayList<>();
-        for (EventDate eventDate : event.getDates()) {
-            dates.add(EventDateDto.builder().startDate(eventDate.getStartDate())
-                .finishDate(eventDate.getFinishDate()).build());
-        }
         EventDto eventDto = new EventDto();
         eventDto.setId(event.getId());
-        eventDto.setDates(dates);
+        eventDto.setTitle(event.getTitle());
         eventDto.setDescription(event.getDescription());
+        eventDto.setTitleImage(event.getTitleImage());
+        eventDto.setOpen(event.isOpen());
+        List<EventDateDto> datesLocations = new ArrayList<>();
+        for (EventDateLocation eventDateLocation : event.getDates()) {
+            EventDateDto eventDateDto = new EventDateDto();
```

Comment:
consider moving this to a separate private method. Also check if stream can be used here.